kore/src/Kore/Builtin/List.hs

```diff
@@ -324,6 +327,15 @@ evalSize resultSort [_list] = do
         & return
 evalSize _ _ = Builtin.wrongArity sizeKey
 
+evalMake :: Builtin.Function
+evalMake resultSort [_len, value] = do
+    _len <- fromInteger <$> Int.expectBuiltinInt getKey _len
+    let len
+          | _len < 0 = 0
```

Comment:
Is there a test of this behavior anywhere, like the frontend integration tests?

Reply:
> Is there a test of this behavior anywhere, like the frontend integration tests?

I don't think so. Should that be an error instead?